tests/acceptance/pageObjects/filesPage.js

```diff
@@ -153,6 +153,21 @@ module.exports = {
         .isVisible(selector, (result) => {
           callback(result.value)
         })
+    },
+
+    /**
+     * Navigates to private link which has been copied to clipboard
+     */
+    navigateToPrivateLink: function () {
+      return this
+        .execute(function () {
+          const input = document.createElement('input')
+          input.setAttribute('id', 'dummy-input-to-get-clipboard')
+        })
```

Comment:
@LukasHirt the second argument for `execute` is an empty array (could not be omitted)

Reply:
Tried to add that as well there and wasn't helping 🤷‍♂